nodejs/nodejs.lua

```diff
@@ -111,8 +136,34 @@ function PLUGIN:Available(ctx)
                 }
             }
         })
+        if major then
+            if not versions_shorthand[major] then
+                versions_shorthand[major] = version
+            else
+                if compare_versions({version = version}, {version = versions_shorthand[major]}) then
+                    versions_shorthand[major] = version
+                end
+            end
+
+            if minor then
+                local major_minor = major .. "." .. minor
+                if not versions_shorthand[major_minor] then
+                    versions_shorthand[major_minor] = version
+                else
+                    if compare_versions({version = version}, {version = versions_shorthand[major_minor]}) then
+                        versions_shorthand[major_minor] = version
+                    end
+                end
+            end
+        end
     end
-    table.sort(result, compare_versions)
+
+    table.sort(list, compare_versions)
+
+    local result = {}
+    result['list'] = list
+    result['versions_shorthand'] = versions_shorthand
+    available_result = result
```

Comment:
You have changed the original behavior. `PLUGIN:Available` needs to return an array instead of a table, and you can use `vfox search nodejs` command to check.

I think the easiest way to do this would be to extract the logic from this function into a separate method.